testing/ef_tests/tests/tests.rs

```diff
@@ -232,7 +232,6 @@ mod ssz_static {
     ssz_static_test!(fork, Fork);
     ssz_static_test!(fork_data, ForkData);
     ssz_static_test!(historical_batch, HistoricalBatch<_>);
-    ssz_static_test!(indexed_attestation, IndexedAttestation<_>);
```

Comment:
we can't generate these tests with superstructed objects, but the full test implementation is included in https://github.com/sigp/lighthouse/pull/5758